dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -63,3 +63,18 @@ accountKeyFile.desc=A path to the file containing a key identifier for a registe
 
 domainKeyFile=Domain key file
 domainKeyFile.desc=A path to the file containing a key identifier for a domain. If the file does not exist, a new key is generated and written to this file. Back this file up to maintain control of the domain. 
+
+acmeTransportConfig=ACME transport configuration
+acmeTransportConfig.desc=ACME transport layer configuration.
+
+sslProtocol=SSL protocol
+sslProtocol.desc=The SSL handshake protocol. Protocol values can be found in the documentation for the underlying JRE's Java Secure Socket Extension (JSSE) provider.  When using the IBM JRE the default value is SSL_TLSv2 and when using the Oracle JRE the default value is SSL.
+
+trustStore=Truststore
+trustStore.desc=A keystore containing trusted certificate entries used by SSL for signing verification.
+
+trustStorePassword=Truststore password
+trustStorePassword.desc=The password used to load the truststore file. The value can be stored in clear text or encoded form. Use the securityUtility tool to encode the password.
```

Comment:
Change "The password used to load the truststore file" to "The password that is used to load the truststore file".

Reply:
Made requested changes.